src/locale/defaults.h

```diff
@@ -628,6 +628,12 @@
 #ifndef D_STR_MIDEA24
 #define D_STR_MIDEA24 "MIDEA24"
 #endif  // D_STR_MIDEA24
+#ifndef D_STR_MILESTAG2SHOT
+#define D_STR_MILESTAG2SHOT "MILESTAG2SHOT"
+#endif  // D_STR_MILESTAG2SHOT
+#ifndef D_STR_MILESTAG2MSG
+#define D_STR_MILESTAG2MSG "MILESTAG2MSG"
+#endif  // D_STR_MILESTAG2MSG
```

Comment:
See other comments/suggestions. Lets just go with a single protocol of `MILESTAG2`.